test/fixtures/books.yml

```diff
@@ -46,4 +46,41 @@ three:
>771215s1977    dcu          f00010 eng d</controlfield>\n    <datafield tag=\"035\" ind1=\" \" ind2=\" \">\n        <subfield code=\"a\">(OCoLC)ocm03493895</subfield>\n    </datafield>\n    <datafield tag=\"035\" ind1=\" \" ind2=\" \">\n        <subfield code=\"9\">ABC-6433</subfield>\n    </datafield>\n    <datafield tag=\"040\" ind1=\" \" ind2=\" \">\n        <subfield code=\"a\">GPO</subfield>\n        <subfield code=\"c\">GPO</subfield>\n        <subfield code=\"d\">VRC</subfield>\n        <subfield code=\"d\">UIU</subfield>\n        <subfield code=\"d\">m.c.2</subfield>\n    </datafield>\n    <datafield tag=\"074\" ind1=\" \" ind2=\" \">\n        <subfield code=\"a\">502-A-2</subfield>\n    </datafield>\n    <datafield tag=\"086\" ind1=\"0\" ind2=\" \">\n        <subfield code=\"a\">HE 23.3102:T 34</subfield>\n    </datafield>\n    <datafield tag=\"110\" ind1=\"2\" ind2=\"0\">\n        <subfield code=\"a\">National Clearinghouse on Aging.</subfield>\n    </datafield>\n    <datafield tag=\"245\" ind1=\"1\" ind2=\"0\">\n        <subfield code=\"a\">National Clearinghouse on Aging thesaurus.</subfield>\n    </datafield>\n    <datafield tag=\"250\" ind1=\" \" ind2=\" \">\n        <subfield code=\"a\">2d ed., July 1977.</subfield>\n    </datafield>\n    <datafield tag=\"260\" ind1=\"0\" ind2=\" \">\n        <subfield code=\"a\">Washington :</subfield>\n        <subfield code=\"b\">Dept. of Health, Education, and Welfare, Office of Human Development Services, Administation on Aging, National Clearinghouse on Aging,</subfield>\n        <subfield code=\"c\">1977.</subfield>\n    </datafield>\n    <datafield tag=\"300\" ind1=\" \" ind2=\" \">\n        <subfield code=\"a\"> 137 p. in various pagings ;</subfield>\n        <subfield code=\"c\">28 cm.</subfield>\n    </datafield>\n    <datafield tag=\"490\" ind1=\"0\" ind2=\" \">\n        <subfield code=\"a\">United States.  Dept. of Health, Education, and Welfare. DHEW publication ;  no. (OHDS) 78-20087</subfield>\n    </datafield>\n    <datafield tag=\"500\" ind1=\" \" ind2=\" \">\n        <subfield code=\"a\">Prepared by Documentation Associates Information Services Incorporated under contract no. HEW 105-76-3000.</subfield>\n    </datafield>\n    <datafield tag=\"650\" ind1=\" \" ind2=\"0\">\n        <subfield code=\"a\">Gerontology</subfield>\n        <subfield code=\"x\">Terminology.</subfield>\n    </datafield>\n    <datafield tag=\"650\" ind1=\" \" ind2=\"0\">\n        <subfield code=\"a\">Aging</subfield>\n        <subfield code=\"x\">Terminology.</subfield>\n    </datafield>\n    <datafield tag=\"955\" ind1=\" \" ind2=\" \">\n        <subfield code=\"a\">UIU</subfield>\n        <subfield code=\"b\">30112048976390</subfield>\n        <subfield code=\"c\">472411</subfield>\n        <subfield code=\"d\">Stacks</subfield>\n        <subfield code=\"e\">301.435014 N213N1977</subfield>\n        <subfield code=\"f\">2</subfield>\n        <subfield code=\"g\">am</subfield>\n    </datafield>\n</record>"
```

Comment:
For the sake of readability, I would probably format this XML and store it in its own file. There is a place for such files in `test/fixtures/files`. Rails fixture YAML files support ERB, so from the books.yml file, you could read in the file like:

`raw_marcxml: <%= File.read(file_fixture("my_marcxml_file.xml")) %>`

Reply:
Updated using a separate fixture file for the xml data